src/test/java/uk/gov/ch/oauth/Oauth2Test.java

```diff
@@ -61,9 +72,21 @@
     @Mock
     private Session session;
     @Mock
+    private Cookie cookie;
+    @Mock
     private SessionFactory sessionFactory;
     @Mock
     private HttpServletRequest request;
+    @Mock(answer = Answers.RETURNS_DEEP_STUBS)
+    private OAuth2StateHandler oAuth2StateHandler;
+    @Mock
+    private Payload payload;
+    @Mock
+    private JSONObject jsonObject;
```

Comment:
Payload and JSONObject are unused.

Reply:
Strange that eclipse isn't showing these as being unused.